src/modules/private-share-folder/private-sharing.controller.ts

```diff
@@ -72,6 +77,50 @@ export class PrivateSharingController {
     }
   }
 
+  @Put('update-role/:id')
```

Comment:
Could we change the naming of this? As a resource on REST, and as we use PUT:
`PUT /role/:id`
would be more correct